accessscopes.go

```diff
@@ -0,0 +1,28 @@
+package goshopify
```

Comment:
I believe it is better to use `_` to split words in file names i.e. change `accessscopes.go` to `access_scopes.go`
Although we already have some examples of not splitting words in the repo, I believe it may be beneficial to split them as most of files we have are using `_` and it is easier to read this way.

Please apply to all changed files

Reply:
Done, thanks for the review!